### PR TITLE
OCD UI Fix

### PR DESCRIPTION
**Pull request by cydia2020**
these have been bothering me for a while so I thought I'd open a PR to address them. 😂

### CODE REVIEW

File: selfdrive/ui/qt/offroad/networking.cc [hunk 1]
```diff
@@ -278,7 +278,7 @@ void WifiUI::refresh() {
       QPushButton *forgetBtn = new QPushButton("FORGET");
       forgetBtn->setObjectName("forgetBtn");
       QObject::connect(forgetBtn, &QPushButton::clicked, [=]() {
-        if (ConfirmationDialog::confirm("Forget WiFi Network \"" + QString::fromUtf8(network.ssid) + "\"?", this)) {
+        if (ConfirmationDialog::confirm("Forget Wi-Fi Network \"" + QString::fromUtf8(network.ssid) + "\"?", this)) {
```
Comment (adeebshihadeh) [inline selfdrive/ui/qt/offroad/networking.cc:281]: I think WiFi looks better. what do you think @pd0wm?

Comment (cydia2020) [reply selfdrive/ui/qt/offroad/networking.cc:281]: > Wi-Fi is a proper noun and a registered trademark. Capitalize and hyphenate when referring specifically to Wi-Fi technologies. Don’t use WiFi, wifi, or Wifi.

https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/w/wi-fi

Comment (jyoung8607) [reply selfdrive/ui/qt/offroad/networking.cc:281]: [Wi-Fi](https://www.wi-fi.org/) is technically correct, the best kind of correct.

Comment (cydia2020) [reply selfdrive/ui/qt/offroad/networking.cc:281]: Yes :)

Comment (adeebshihadeh) [reply selfdrive/ui/qt/offroad/networking.cc:281]: alright

**Pull request merged**